Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml

```diff
@@ -18,6 +18,74 @@ Neos:
           host: 127.0.0.1
           port: 6379
 
+  ContentRepositoryRegistry:
+    contentRepositories:
+      test_parallel:
+        eventStore:
+          factoryObjectName: Neos\ContentRepositoryRegistry\Factory\EventStore\DoctrineEventStoreFactory
+
+        nodeTypeManager:
+          factoryObjectName: Neos\ContentRepository\BehavioralTests\TestSuite\Behavior\GherkinPyStringNodeBasedNodeTypeManagerFactory
```

Comment:
the `GherkinPyStringNodeBasedNodeTypeManagerFactory` thingy should be renamed to some generic `StaticNodeTypeManagerFactory` so it can be used here without weirdneess

Reply:
will be done in a followup: https://github.com/neos/neos-development-collection/pull/5346